README.md

```diff
@@ -570,6 +570,28 @@ func main() {
 }
 ```
 
+### Audit Log (Warden)
```

Comment:
You can use [`doctoc README.md`](https://www.npmjs.com/package/doctoc) to update the TOC